packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -110,6 +110,12 @@ export function createLodestarMetrics(
         help: "Current count of pending items in reqRespBridgeReqCaller data structure",
       }),
     },
+    networkWorkerWireEventsOnMainThreadLatencySec: register.histogram<"eventName">({
+      name: "lodestar_network_worker_wire_events_on_main_thread_latency_seconds",
```

Comment:
While looking at networking metrics I noticed that the name here does not match with the value used in the dashboard

https://github.com/ChainSafe/lodestar/blob/e42d6cc6457051cf67ab7a3cdbfada25028d10f7/dashboards/lodestar_networking.json#L940